depth_width: 640
depth_height: 360
enable_depth: true
color_width: 640
color_height: 360
enable_color: true

# depth_width: 848
# depth_height: 480
# enable_depth: true
# color_width: 848
# color_height: 480
# enable_color: true

# depth_width: 1280
# depth_height: 720
# enable_depth: true
# color_width: 1280
# color_height: 720
# enable_color: true